lib/bashcov.rb

```diff
@@ -60,17 +83,25 @@ def help(program_name)
 
     def option_parser
       OptionParser.new do |opts|
-        opts.program_name = "bashcov"
+        opts.program_name = program_name
         opts.version = Bashcov::VERSION
         opts.banner = help opts.program_name
 
         opts.separator "\nSpecific options:"
 
         opts.on("-s", "--skip-uncovered", "Do not report uncovered files") do |s|
-          @options.skip_uncovered = s
+          options.skip_uncovered = s
         end
         opts.on("-m", "--mute", "Do not print script output") do |m|
-          @options.mute = m
+          options.mute = m
+        end
+        opts.on("--bash-path PATH", "Path to Bash executable") do |p|
+          raise Errno::ENOENT, p unless File.file? p
+          options.bash_path = p
+        end
+        opts.on("--root PATH", "Project root directory") do |d|
```

Comment:
I found this a useful shortcut when making changes to Bashcov and testing them against shell scripts in other directories.  But that's a pretty slender use case!  TBH the reason I added this was because I needed to make the project root configurable in order to properly test the temporary scripts created by the test suite, and I figured that there'd be little harm in exposing the setting as a command line option.